doc/changelog/10-standard-library/11725-cleanup-reals.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  Use implicit arguments for ConstructiveReals. Move ConstructiveReals
```

Comment:
```suggestion
Use implicit arguments for ``ConstructiveReals``. Move ``ConstructiveReals``
```

By "Use implicit arguments", you mean that the `R:ConstructiveReals` argument of lemmas is now implicit, right. Maybe, this can be told more explicitly.